.changes/operator/v2.3.7-24.3.6.md

```diff
@@ -0,0 +1,30 @@
+## v2.3.7-24.3.6 - 2025-02-28
+### Changed
+* Bumped internal redpanda chart to  v5.9.19.
```

Comment:
Miss on my behalf but it'd be nice to clip this out as the next bullet point in validates it. We can rip this out retroactively though so as to not reset the CI timer.